lib/domain/entity/movie.dart

```diff
@@ -71,4 +71,16 @@ class Movie {
           'brothers are separated, Mario embarks on an epic quest to find Luigi.',
     );
   }
+
+  @override
+  bool operator ==(Object other) =>
+      identical(this, other) ||
+      other is Movie &&
+          runtimeType == other.runtimeType &&
+          voteCount == other.voteCount &&
+          id == other.id &&
+          voteAverage == other.voteAverage;
+
+  @override
+  int get hashCode => voteCount.hashCode ^ id.hashCode ^ voteAverage.hashCode;
```

Comment:
Why you need to define this?

Reply:
i defined this, for testing purposes. To be able to compare two objects, in this case movies.